### DISCUSSION TITLE
台灣柑仔店在紐約，透過果乾、台灣食物找回歸屬感、推廣文化

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1133] [General]
https://www.thenewslens.com/article/178658
我覺得雲海群眾募資結果意料之外熱烈，最終募到近11萬美元（約新台幣338萬元）。他們透過美國發起群眾募資，直接向台灣小農採購做成果乾，以利進口到美國，我認為十分聰明。雲海希望形成永久合作關係，讓雲海能一直把台灣的水果介紹給美國人。

**Comment by woohsin** [comment]
這種永久合作關係的建立，將有助於加深台美之間的友好關係，並促進兩國在農業和食品領域的合作。這對於台灣來說是一個很好的機會，可以將台灣的農產品推廣到國際市場，提高台灣農業的國際競爭力。

總的來說，這個募資活動的成功不僅對雲海公司和台灣小農是一個好消息，也對整個台灣農業和對外貿易是一個積極的信號。
